api/src/main/java/org/hawkular/apm/api/model/Constants.java

```diff
@@ -43,4 +45,46 @@
      */
     public static final String DETAIL_FAULT_DESCRIPTION = "fault.description";
 
+    /**
+     * Represents database component type of
+     * {@link org.hawkular.apm.api.model.trace.Component#componentType}
+     */
+    public static final String COMPONENT_DATABASE = "Database";
+
+    /**
+     * Represents EJB component type of
+     * {@link org.hawkular.apm.api.model.trace.Component#componentType}
+     */
+    public static final String COMPONENT_EJB = "EJB";
+
+    /**
+     * Binary annotation key for HTTP URL.
+     *
+     * The entire URL, including the scheme, host and query parameters if available. See zipkinCoreConstants.
+     */
+    public static final String ANNOTATION_HTTP_URL = "http.url";
```

Comment:
Just wondering whether we should place zipkin specific constants in a separate file? Or have a section of the file clearly marked as zipkin definitions?